### PR TITLE
Add test_transaction.py

### PR DESCRIPTION
**Pull request by esganesh**

### CODE REVIEW

File: tests/test_transaction.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        call_module='Balances',
+        call_function='transfer',
+        call_params={
+            'dest': BOB,
+            'value': 1 * 10 ** 12
+        }
+    )
+
+    # Get payment info
+    payment_info = kagome.get_payment_info(call=call, keypair=keypair_alice)
```
Comment (adaagava) [inline tests/test_transaction.py:49]: Why not
payment_info = kagome.get_payment_info(call, keypair_alice)

?

File: tests/test_transaction.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    bob_balance_new = bob_kagome_new.get('data').get('free', 0) / 10 ** 12
+
+    logging.info('Alice free balance: %s', free_alice_balance)
+    logging.info('Alice balance after transaction: %s', alice_balance_new)
+
+    logging.info('Bob free balance: %s', free_bob_balance)
+    logging.info('Bob balance after transaction: %s', bob_balance_new)
+
+    assert_that(alice_balance_new).is_less_than_or_equal_to(free_alice_balance - 1)
+    assert_that(bob_balance_new).is_greater_than_or_equal_to(free_bob_balance + 1)
```
Comment (adaagava) [inline tests/test_transaction.py:88]: when you have several asserts in one test, please add descriptions
assert_that(alice_balance_new.is_less_than_or_equal_to(free_alice_balance - 1), 'Alice balance is bigger than it was before transaction!')

**Review by adaagava — CHANGES_REQUESTED** [review]
Great job! Just some small cosmetic changes needed

File: tests/test_transaction.py (hunk 1 above)
Comment (esganesh) [reply tests/test_transaction.py]: It's as in official example. But I checked - It works as your wrote
Done

File: tests/test_transaction.py (hunk 2 above)
Comment (esganesh) [reply tests/test_transaction.py]: Done

File: tests/test_transaction.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+        url=KAGOME_S1,
+        type_registry_preset='development'
+    )
+
+    substrate = SubstrateInterface(
+        url=SUBSTRATE_S2,
+        type_registry_preset='development'
+    )
+
+    keypair_alice = Keypair.create_from_uri('//Alice')
```
[the remark this answers is not in the document]
Comment (esganesh) [reply tests/test_transaction.py:22]: @kamilsa ?

File: tests/test_transaction.py [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+    alice_balance_new = alice_kagome_new.get('data').get('free', 0) / 10 ** 12
+    bob_balance_new = bob_kagome_new.get('data').get('free', 0) / 10 ** 12
+
+    logging.info('Alice free balance: %s', free_alice_balance)
+    logging.info('Alice balance after transaction: %s', alice_balance_new)
+
+    logging.info('Bob free balance: %s', free_bob_balance)
+    logging.info('Bob balance after transaction: %s', bob_balance_new)
+
+    assert_that(alice_balance_new).described_as("Alice's balance didn't decrease after transaction").is_less_than_or_equal_to(free_alice_balance - 1)
```
Comment (esganesh) [inline tests/test_transaction.py:87]: @kamilsa 
Do we need check Alice's balance more detailed?
<Balance after transaction> = <Balance before transaction> - 1 - Fee

or it's enough
<Balance after transaction> <= <Balance before transaction> - 1 ?

File: tests/test_transaction.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (kamilsa) [reply tests/test_transaction.py:22]: this is standard keys, we may keep them for every run

File: tests/test_transaction.py (hunk 4 above)
Comment (kamilsa) [reply tests/test_transaction.py:87]: no need. <= -1 is enough

**Review by kamilsa — APPROVED** [review]
Maybe pip.lock file is not needed to commit and could be put into .gitignore. Otherwise looks good

File: tests/test_transaction.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (adaagava) [reply tests/test_transaction.py:22]: So please move them to fixtures later

**Review by adaagava — APPROVED** [review]

**Comment by esganesh** [comment]
"Maybe pip.lock file is not needed to commit and could be put into .gitignore. Otherwise looks good"

https://pipenv.pypa.io/en/latest/basics/
Tutorials advice "Generally, keep both Pipfile and Pipfile.lock in version control." "Do not keep Pipfile.lock in version control if multiple versions of Python are being targeted."

**Pull request merged**